modules/tutorials/pages/java-tutorial/retrieve-records.adoc

```diff
@@ -0,0 +1,275 @@
+= Retrieve Records
+:description: Retrieve records, or documents, from your collections using {sqlpp}, Couchbase's SQL-based query language.
+:page-topic-type: tutorial
+:page-pagination: full
+:page-toclevels: 2
+:imagesdir: ../../images
+:page-aliases: retrieving-documents.adoc
+
+[abstract]
+{description}
+You can retrieve your records using the <<#retrieve-with-query-editor,query editor>> or the <<#retrieve-with-sdk,SDK>>.
+
+[#retrieve-with-query-editor]
+== With the Query Editor
+
```

Comment:
I think Vale should have flagged the stacked headings - can you think of even a single sentence to add here so we don't have two headings with nothing in between?

Reply:
Maybe an overview of the substeps, like: 

To retrieve the records with the Query Editor, you must first define an index.